### PR TITLE
updated moes switch convertors

### PR DESCRIPTION
**Pull request by vladispavlov**
Previous pr: https://github.com/Koenkk/zigbee-herdsman-converters/pull/3822
Not sure about https://github.com/Koenkk/zigbee-herdsman-converters/commit/6028a32ad8fbae02796e9611873104730337b397. Maybe there is a better way to fix this.

Sorry, I haven't fully tested the previous converter. Have fixed bugs and now it's ok.

### CODE REVIEW

File: converters/fromZigbee.js [hunk 1]
```diff
@@ -2420,6 +2420,12 @@ const converters = {
         convert: (model, msg, publish, options, meta) => {
             const dpValue = tuya.firstDpValue(msg, meta, 'moes_switch');
             const dp = dpValue.dp;
+
+            // tuya_switch datapoints
+            if (dp >= 1 && dp <= 4) {
```
Comment (Koenkk) [inline converters/fromZigbee.js:2425]: Why was this added, this prevents `fromZigbee:moes_switch: NOT RECOGNIZED DP` from being logged.

Comment (vladispavlov) [reply converters/fromZigbee.js:2425]: If not to add this condition when the state changes it also displays warning `fromZigbee:moes_switch: NOT RECOGNIZED DP`. So `tuya_switch` and `moes_switch` converters work together at the same time.
I'm sure that it's solution not good, but it works fine. Maybe it makes sense to replace switch case to if conditions.

Comment (Koenkk) [reply converters/fromZigbee.js:2425]: thanks for the explanation!

**Comment by SimonMerrett** [comment]
Sorry if this is hijacking but the fact that this PR is fixing things makes me wonder if I am able to change something to enable binding of a Sonoff snzb-01 button to a Moes ZTS-EU_1gang light switch. I am able to bind the Sonoff buttons to Philips Hue bulbs but whenever I try to bind (command line and browser front end) I get 
```
Zigbee2MQTT:error 2022-02-07 18:33:12: Nothing to bind from 'sonoff_switch_13' to 'moes_switch_01'
Zigbee2MQTT:info  2022-02-07 18:33:12: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":{"from":"sonoff_switch_13","to":"moes_switch_01"},"type":"device_bind_failed"}'
Zigbee2MQTT:info  2022-02-07 18:33:12: MQTT publish: topic 'zigbee2mqtt/bridge/response/device/bind', payload '{"data":{"clusters":[],"failed":[],"from":"sonoff_switch_13/1","to":"moes_switch_01/1"},"error":"Nothing to bind","status":"error","transaction":"on6wv-9"}'
Zigbee2MQTT:error 2022-02-07 18:33:12: Nothing to bind
```
Is there something in the converters that should be changed to let the Sonoff button bind with the Moes light switch?
